src/qae/object-definition.js

```diff
@@ -176,6 +176,12 @@ const objectDefinition = () => {
        * @default 5
        */
       rangeBubbleSizes: [2, 8],
+      /**
+       * Resolution settings for compressed data.
+       * @type {number}
+       * @default 6
+       */
+      queryLevel: 6,
```

Comment:
This should be `compressionResolution`

Reply:
yes, will do that. I misunderstood you on slack.